app/players/dwarves/dwarf_blacksmith.py

```diff
@@ -0,0 +1,23 @@
+from app.players.dwarves.dwarf import Dwarf
+
+
+class DwarfBlacksmith(Dwarf):
+    def __init__(
+            self,
+            nickname: str,
+            favourite_dish: str,
+            skill_level: int
+    ) -> None:
+        super().__init__(nickname, favourite_dish)
+        self._skill_level = skill_level
+
+    def get_rating(self) -> int:
+        return self._skill_level
+
+    def player_info(self) -> str:
+        return (f"Dwarf blacksmith "
+                f"{self.nickname} with skill of the "
+                f"{self._skill_level} level")
+
+    def eat_favourite_dish(self) -> None:
+        print(f"{self.nickname} is eating {self._favourite_dish}")
```

Comment:
This method is duplicated. You can remove it and implement it once in Dwarf class